glusterd2/config.go

```diff
@@ -86,6 +87,10 @@ func setDefaults() error {
 		config.SetDefault("localstatedir", wd)
 	}
 
+	if config.GetString("hooksdir") == "" {
+		config.SetDefault("hooksdir", wd+"/hooks/1")
```

Comment:
This should default to `localstatedir` which is equivalent to `/var/lib/glusterd`.

What does the `1` refer to ? API version ?

Reply:
> What does the 1 refer to ? API version ?

Hooks API version used in Glusterd1(I think this versioning is based on the parameters sent to hook scripts, if parameters changes then version increment is required)